packages/persist/lib/routes/environment/environmentId/connection/connectionId/getCursor.ts

```diff
@@ -3,6 +3,7 @@ import { validateRequest } from '@nangohq/utils';
 import type { EndpointRequest, EndpointResponse, RouteHandler, Route } from '@nangohq/utils';
```

Comment:
just realized since we are passing the secretKey `/environement/${id}` could be removed

Reply:
yes, it is nice for debugging to filter by `url` though